yarn-project/end-to-end/src/e2e_crowdfunding_and_claim.test.ts

```diff
@@ -95,7 +95,10 @@ describe('e2e_crowdfunding_and_claim', () => {
       deadline,
     );
     const crowdfundingInstance = await crowdfundingDeployment.getInstance();
-    await wallet.registerAccount(crowdfundingSecretKey, await computePartialAddress(crowdfundingInstance));
+    await wallet.registerKeysForEscrowContract(
+      crowdfundingSecretKey,
+      await computePartialAddress(crowdfundingInstance),
+    );
```

Comment:
Renamed the problematic function to make it less cancerous before it gets completely nuked.